.github/workflows/pythonapp.yml

```
@@ -0,0 +1,35 @@
+name: Syrupy CICD
+
+on: [push]
+
+env:
+  CI: 1
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install python dev dependencies.
+      run: |
+        python -m pip install -U pip --no-cache-dir
+        python -m pip install pip-tools wheel
+        python -m pip install -e . -r requirements.txt -r dev-requirements.txt
+    - name: Lint
+      run: |
+        invoke lint
+    - name: Test
+      run: |
+        invoke test
+    - name: Build Distribution
+      run: |
+        python setup.py sdist bdist_wheel
```

Could probably make an invoke command

---

Should also stub out a release step here
```suggestion
        inv build
    - name: Publish
      run: |
        inv publish
```

---

I'll do this in a follow up